roles/module_openapi_cloud/tasks/main.yaml

```diff
@@ -24,14 +24,11 @@
   when: ( plugin['action'] == 'generate_examples' ) or ( plugin['action'] == 'generate_all' )
 
 - name: Generate ignore files for "{{ collection['name'] }}"
-  ansible.builtin.command: python3 "{{ role_path }}"/templates/refresh_ignore_files.py
-    --target-dir "{{ collection['path'] }}"
-  delegate_to: 127.0.0.1
-  run_once: true
+  ansible.content_builder.generate_cloud_ignore_files:
+      target_dir: "{{ collection['path'] }}"
   changed_when: false
   when:
       - ( plugin['action'] == 'generate_ignore_files' ) or ( plugin['action'] == 'generate_all' )
-      - plugin['name'] == 'vmware_rest'
```

Comment:
I think this should not be removed.

Reply:
with `- plugin['name'] == 'vmware_rest'` it skips the task to `generate_cloud_ignore_files` entirely, not sure if I'm missing something but looks like that condition should be either removed or changed to `- plugin['name'] == 'amazon_cloud'`

Reply:
refresh_ignore_files.py is applicable only for vmware_rest. amazon_cloud does not need this plugin.

Reply:
Added.